framework/core/notify/webhook.py

```
@@ -0,0 +1,32 @@
+    def notify_verification(self, org_name: str, org_domain: str, email: str, token: str) -> None:
+        if not self.is_configured():
+            return
+        payload = {
+            "type": "org_verification",
+            "org_name": org_name,
+            "org_domain": org_domain,
+            "email": email,
+            "token": token,
+        }
+        data = json.dumps(payload).encode("utf-8")
+        req = urllib.request.Request(self.url, data=data, headers={"Content-Type": "application/json"})
```

Ruff (0.13.2)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)